blog/unified-plans.md

```diff
@@ -0,0 +1,103 @@
+A: We're simplifying our plan offerings to streamline operations, ensure fair resource usage, and support sustainable growth. This change will help us deliver a more efficient and agile service to our community.
+
+**Q: What are the new plans available?**  
+A: Starting March 13th, 2024, we'll offer four plans: Netdata Enterprise, Netdata Business, Netdata Homelab, and Netdata Community. Each is designed to meet the specific needs of different user segments, from large organizations to personal, non-professional use.
+
+**Q: What will happen to my current Early Bird or Unlimited Community Plan?**  
+A: If you're on one of these plans, you'll be automatically transitioned to a 60-day free trial of the Netdata Business plan. After the trial, you can choose the plan that best suits your needs.
+
+**Q: What if I don't choose a new plan after the trial?**  
+A: If you don't select a new plan, you will automatically be moved to the Netdata Community Plan, which includes usage limits. However, these limits are designed to accommodate the needs of many of our users comfortably.
+
+**Q: Will there be any cost to me after the change?**  
+A: It depends on your usage and the plan you select after the trial period. We're offering a 25% lifetime discount on the Business plan for those affected by the change, applicable for annual plans. If your usage fits within the new Community Plan's limits, there may be no cost to you.
+
+**Q: Can I keep monitoring the same number of nodes?**  
+A: The ability to concurrently monitor the same number of nodes will depend on the plan you choose after the trial. The Community Plan has limits on concurrently monitored nodes, but the Homelab, Business, and Enterprise plans are designed to accommodate monitoring at any scale.
+
+**Q: What are the limits on the community plan?**
+A: The community plan has a limit of 5 nodes that can be concurrently monitored and visualized on the Netdata dashboards. You can have an unlimited number of nodes connected to Netdata Cloud, however you will need to choose which 5 nodes you want to visualize at a given time.
+
+**Q: How do I choose a new plan or take advantage of the discount?**  
+A: You can select a new plan or apply the discount by visiting the Plans and Billing section in the Netdata App. Enter the provided coupon code at checkout to apply the discount.
+
+**Q: What if I have more concerns, questions or need assistance choosing a plan?**  
+A: We're here to help. Please reach out to us at plans@netdata.cloud with any questions or for guidance on selecting the plan that's right for you.
+
+**Q: How does this affect my data and the monitoring I currently have set up?**  
+A: Your data and current monitoring setups will not be affected during the transition to the new plan structure. We aim to make this process as seamless as possible for all our users.
+
+**Q: I am already a Netdata business or homelab user, how does this change affect me?**
+A: You will NOT be affected in any way.
```

Comment:
not sure why this doesn't apply the newline
![image](https://github.com/netdata/blog/assets/82235632/0cae9b65-ba09-41f0-b37c-bb17ecd9f280)

Reply:
This is now fixed.